### DISCUSSION TITLE
Curious to know Tailwind got JIT what is the motivation for windicss now?

### DISCUSSION THREAD

**Discussion by Chetan11-dev** [windicss/windicss#724] [General]
Curious to know Tailwind got JIT what is the motivation for maintaining windicss now? #724

**Comment by antfu** [comment]
https://github.com/windicss/windicss/discussions/176

**Comment by Chetan11-dev** [comment]
How does it differentiate from tailwindcss in terms of user experience to me as a developer?

**Comment by quick007** [comment]
> How does it differentiate from tailwindcss in terms of user experience to me as a developer?

Tailwind mostly just works. Run `yarn add tailwindcss` and `npx tailwindcss init` and you're off to the races. Windi CSS seems to not have framework-specific plugins for many mainstream frameworks, so you'll have to do some custom configuration. Tailwind also has an ecosystem of first and third party plugins built around it. 

On the other hand, Windi has a handful of other styles and devtools style editing support. They also mention they aren't part of a company (tailwind is made by tailwind labs which also makes Heroicons and tailwind UI, complementary software to Tailwind.) Also if you care, some people use Windi out of spite for tailwind because tailwind didn't mention Windi was the inspiration for JIT until someone mentioned it.
